src/core/core.scale.js

```diff
@@ -704,7 +715,7 @@ module.exports = function(Chart) {
 					var isLeft = options.position === 'left';
 					scaleLabelX = isLeft ? me.left + (scaleLabelFont.size / 2) : me.right - (scaleLabelFont.size / 2);
 					scaleLabelY = me.top + ((me.bottom - me.top) / 2);
-					rotation = isLeft ? -0.5 * Math.PI : 0.5 * Math.PI;
+					rotation = -0.5 * Math.PI;
```

Comment:
Please compare these two examples.
![2016-12-30 2 _li](https://cloud.githubusercontent.com/assets/22541770/21554994/21c67baa-ce4f-11e6-8982-28dc6d828cc0.jpg)
![2016-12-30 1 _li](https://cloud.githubusercontent.com/assets/22541770/21554989/1d9caee6-ce4f-11e6-8282-79905afa479e.jpg)

I think that the second figure is easier to read when I look at the four axis labels.
Because It is a personal opinion, I think that it is OK to cancel this change.